src/types/tokens.rs

```diff
@@ -88,11 +151,25 @@ impl std::fmt::Display for FTBalance {
     }
 }
 
+/// Account balance on the NEAR blockchain.
+///
+/// This balance doesn't include staked NEAR tokens or storage
 #[derive(Debug, Clone, PartialEq, Default, Eq, Serialize, Deserialize)]
 pub struct UserBalance {
-    pub liquid: NearToken,
-    pub locked: NearToken,
+    /// The total amount of NEAR tokens in the account.
+    ///
+    /// Please note that this is the total amount of NEAR tokens in the account, not the amount available for use.
+    pub total: NearToken,
+    /// The amount of NEAR tokens locked in the account for storage usage.
+    pub storage_locked: NearToken,
```

Comment:
it can be added that storage_locked is equal to storage_usage * STORAGE_PER_BYTE at all times, 
if this info can be confirmed to be true in all cases 100%

Reply:
I believe, there is other locks, so that why I called it storage_locked